src/test/java/nearsoft/academy/bigdata/recommendation/MovieRecommenderTest.java

```diff
@@ -24,7 +33,5 @@ public void testDataInfo() throws IOException, TasteException {
         assertThat(recommendations, hasItem("B0002O7Y8U"));
         assertThat(recommendations, hasItem("B00004CQTF"));
         assertThat(recommendations, hasItem("B000063W82"));
-
```

Comment:
Is usually a good idea to have just the necessary changes on the PR's,  we should try not to include format changes unless is required. Avoiding them makes it easier to review the PR's.

Reply:
Thanks for the advice! I'll restore the original file with only the necessary changes.